src/app/Http/Controllers/Admin/ArticleCrudController.php

```diff
@@ -150,6 +150,7 @@ public function setup()
                 'name' => 'status',
                 'label' => 'Status',
                 'type' => 'enum',
```

Comment:
Maybe we should just turn this into a `select_from_array`? That would solve the problem for all DBMSs, won't it? I don't think we have a reason to be fancy here and use `enum` - might have been a bad first choice...

What do you think @promatik ?

Reply:
@tabacitu makes total sense, I've changed it 🙌